en/SQLDatabase.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
What is this? Please double check that you have a file WITHOUT [UTF8-BOM](https://de.wikipedia.org/wiki/Byte_Order_Mark).